api/scripts/data-generation/generate-certifications-for-organization.js

```diff
@@ -269,19 +270,18 @@ async function _createCertificationResultsWithMarks({ assessmentIds, status, pix
   for (const competence of pixCompetences) {
     await _createCompetenceMarksForCompetence({ competence, assessmentResultIds, transaction });
   }
-
   await transaction.raw(`
   WITH sum_score AS (
     SELECT DISTINCT "assessmentResultId",
            SUM("score") OVER (PARTITION BY "assessmentResultId") AS final_score
     FROM "competence-marks"
-    WHERE "assessmentResultId" = ANY(ARRAY[?])
+    WHERE "assessmentResultId" = ANY(ARRAY[${assessmentResultIds.join(',')}])
```

Comment:
Ugly fix (qui ne passe pas le linter) parce que l'interpolation avec le `?` semble péter une exception... 
Deux messages : 

- `operator does not exist: integer = text`
- `No operator matches the given name and argument types. You might need to add explicit type casts`